### ISSUE TITLE
Wow, that is quite an effort.

### ISSUE THREAD

**Issue by antoan** [mpizenberg/multiple-view-geometry#1]
Thanks for this, really appreciate it since I could not get the lecture slides from the official website - requires a password. Also the videos saturate in brightness at times and it is hard to follow the math.

**Comment by mpizenberg** [comment]
You're welcome, it is not complete though, missing the last few videos. There might also be some errors in the rewrite. Contributions welcomed if you spot anything!
